src/UI/Component/ViewControl/Factory.php

```diff
@@ -64,4 +64,38 @@ public function mode($labelled_actions, $aria_label);
 	 */
 	public function section(Button $previous_action, Button $button, Button $next_action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The sortation view control enables users to change the order in which
+	 *      some data is presented.
+	 *      The common table-sorting mechanism, e.g., in which a column header is clicked,
+	 *      requires the column to be visible. Visibility and sorting can be decoupled
+	 *      by a control next to the actual table.
+	 *      This concept is used by many pages, notably in shops like e.g. amazon, and can
+	 *      be applied to not only tables but all kinds of data-listings.
+	 *   composition: >
+	 *      Sortation uses a Dropdown to display a collection of shy-buttons.
+	 *   effect: >
+	 *      A click on an option will change the ordering of the associated data-list
+	 *      by calling the current page with the parameter "sortation" and its
+	 *      value according to the selected option.
+	 *      The label displayed in the dropdown will be set to the selected sorting.
+	 *
+	 * rules:
+	 *   usage:
+	 *      1: A Sortation MUST NOT be used standalone.
+	 *      2: Sortations MUST BE visually close to the list or table their operation will have effect upon.
+	 *      3: There SHOULD NOT be more than one Sortation per page.
+	 *   accessibility:
+	 *      1: Sortation MUST be operable via keyboard only.
+	 *
+	 * ---
+	 * @param array<string,string>  $options 	a dictionary with value=>title
+	 *
+	 * @return \ILIAS\UI\Component\ViewControl\Sortation
+	 */
+	public function sortation(array $options);
```

Comment:
Basically it is really similar to a click of a button. You do not need to bind some sorting to every individual shy button in the dropdown, there can be one general trigger:
```
$sortable = buildSortableComponent();
$sortation = buildSortationComponent():
$sortation->withOnSort($sortable->getSortSignal());
```
The sortation can internally bind code to the shy button to generate the appropriate js-code. Does that sound plausible?